packages/benchmarks/generator/package.json

```diff
@@ -14,8 +14,8 @@
   "scripts": {
     "build": "tsc",
     "build:watch": "tsc --watch",
-    "serve": "es-dev-server --node-resolve --dedupe --preserve-symlinks",
-    "serve:open": "es-dev-server --node-resolve --dedupe --preserve-symlinks --open generated/index.html"
+    "serve": "web-dev-server --node-resolve --dedupe --preserve-symlinks",
+    "serve:open": "web-dev-server --node-resolve --dedupe --preserve-symlinks --open generated/index.html"
```

Comment:
did you double-check all the flags? I _think_ they're the same

Reply:
They are the same except `dedupe` has been removed.  I can remove.